src/main/java/pb360/service/impl/GenerateRestApiServiceImpl.java

```diff
@@ -37,8 +60,22 @@ public MessageObject getRestApiData(String restId) {
 		getMessage.setType("GET");
 		getMessage.setDatetime(getCurrentDateTime().getDatetime());
 
+		@SuppressWarnings("resource")
```

Comment:
what the resource? what are you doing on this code, we have a application.properties to provide this configuration, why you need to add more configuration at here? please contact Phuoc about how to read data from DB, do not copy a code from the internet then parse to project if you don't understand

Reply:
This was added from suggestion by Eclipse (Resource leak: 'mongoClient' is never closed)